patterns/2-structured/governance-levels.md

```diff
@@ -53,42 +56,59 @@ The same levels make sense inside of organizations.
 
 The solution is to create a universally understood language to describe the InnerSource operating models that are used in your organization.
```

Comment:
Could we replace "operating model" with "governance level" everywhere, to use this term consistently through out the pattern?